webapp-django/crashstats/cron/management/commands/cronrun.py

```diff
@@ -148,6 +148,8 @@ def _run_one(self, job_spec, force=False, cmd_args=None):
                         # the last_success as an argument
                         cmd_kwargs['last_success'] = format_datetime(last_success)
 
+                    logger.info('running: %s %s %s', cmd, cmd_args, cmd_kwargs)
```

Comment:
This makes it easier to see all the actual run lines so we know which commands got which arguments.